docs/src/examples/introduction-to-hdc.jl

```diff
@@ -24,17 +24,23 @@ BipolarHV()
 # value in `HyperdimensionalComputing.jl`, but one can can create a hypervector of any given
 # dimensionality by providing the size of this as an argument:
 
-BipolarHV(8)
+BipolarHV(; dims = 8)
 
-# Alternatively, one can create a hypervector directly from a `AbstractVector`:
+# Alternatively, one can create a hypervector directly from a `Vector{T}` where `{T}` is an
+# appropiate data type, e.g. booleans for BipolarHV:
 
-BipolarHV(rand([-1, 1], 8))
```

Comment:
Not a big fan of this. Bipolar should be [-1, 1]. The user should not be aware that this is stored as a bitvector...